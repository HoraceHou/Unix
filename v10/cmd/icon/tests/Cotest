. ../Setpath
$Icont -s cotest.icn -x
rm -f cotest
