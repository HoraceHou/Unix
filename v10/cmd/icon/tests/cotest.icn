procedure main()
   @create 1
   write("co-expression context switch is implemented")
end
