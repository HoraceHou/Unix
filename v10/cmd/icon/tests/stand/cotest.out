co-expression context switch is implemented
